### PR TITLE
ZEN-10017: Add README.md files to storybook, bootstrap and button packages

### PR DESCRIPTION
**Pull request by leferreyra**

### CODE REVIEW

File: packages/button/package.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
   "version": "0.1.6",
-  "description": "A generic button component",
+  "description": "A generic button Vue.js component",
   "main": "dist/index.js",
   "scripts": {},
   "author": "",
   "license": "ISC",
   "peerDependencies": {
+    "@reciprocity/bootstrap": "0.x",
+    "bootstrap": "4.x",
```
Comment (mplesec) [inline packages/button/package.json]: Should we lock versions?

File: packages/button/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+yarn add @reciprocity/button
+```
+
+## Usage
+
+This component is meant to be used as an ES module, so you can import it using the ES6 `import` syntax.
+
+After importing you can just register it with Vue. Eg:
+
+app.vue
```
Comment (mplesec) [inline packages/button/README.md]: NIT: Rather specify the file as a comment inside the code block, i.e.:
```
// file.vue

code ...
```

File: packages/button/package.json (hunk 1 above)
Comment (mzgajner) [reply packages/button/package.json]: That's not a bad idea, but then we should loosen them (or get rid of them) in ZenGRC.

Comment (leferreyra) [reply packages/button/package.json]: Have in mind now NPM doesn't really do anything for peer deps. This is only to show a message when installing the package.

**Pull request merged**